### PR TITLE
[Merged by Bors] - feat(MeasureTheory/Integral): hasSum_integral_of_summable_integral_norm

### PR DESCRIPTION
**Pull request by loefflerd**
A lemma on interchanging sums and integrals (pretty much equivalent to`integral_tsum`, but without explicitly mentioning `ENNReals`, `lintegrals` etc). Also, correct a random capitalisation glitch I spotted in the process.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Integral/Bochner.lean [hunk 1]
```diff
@@ -1563,6 +1563,20 @@ theorem integral_tsum {ι} [Countable ι] {f : ι → α → G} (hf : ∀ i, AES
     exact hx.of_norm.hasSum
 #align measure_theory.integral_tsum MeasureTheory.integral_tsum
 
+lemma hasSum_integral_of_summable_integral_norm {ι} [Countable ι] (F : ι → α → E)
+    (hF_int : ∀  i : ι, Integrable (F i) μ) (hF_sum : Summable fun i ↦ ∫ a, ‖F i a‖ ∂μ) :
+    HasSum (fun n ↦ ∫ (a : α), F n a ∂μ) (∫ (a : α), (∑' (i : ι), F i a) ∂μ) := by
+  rw [integral_tsum (fun i ↦ (hF_int i).1)]
+  · exact (hF_sum.of_norm_bounded _ fun i ↦ norm_integral_le_integral_norm _).hasSum
+  have (i : ι) : ∫⁻ (a : α), ‖F i a‖₊ ∂μ = ‖(∫ a : α, ‖F i a‖ ∂μ)‖₊
+  · rw [lintegral_coe_eq_integral _ (hF_int i).norm, coe_nnreal_eq, coe_nnnorm,
```
Comment (sgouezel) [inline Mathlib/MeasureTheory/Integral/Bochner.lean:1572]:
```suggestion
  have (i : ι) : ∫⁻ (a : α), ‖F i a‖₊ ∂μ = ‖(∫ a : α, ‖F i a‖ ∂μ)‖₊ := by
    rw [lintegral_coe_eq_integral _ (hF_int i).norm, coe_nnreal_eq, coe_nnnorm,
```
The syntax you were using is a Lean 3-ism, soon to be dropped in Lean 4.

File: Mathlib/MeasureTheory/Integral/Bochner.lean [hunk 2]
```diff
@@ -1563,6 +1563,20 @@ theorem integral_tsum {ι} [Countable ι] {f : ι → α → G} (hf : ∀ i, AES
     exact hx.of_norm.hasSum
 #align measure_theory.integral_tsum MeasureTheory.integral_tsum
 
+lemma hasSum_integral_of_summable_integral_norm {ι} [Countable ι] (F : ι → α → E)
```
Comment (sgouezel) [inline Mathlib/MeasureTheory/Integral/Bochner.lean:1566]: could you also write a lemma `integral_tsum'` under your assumptions? (it is a direct consequence of the one you've just written)

**Comment by loefflerd** [comment]
Thanks for the comments, I've uploaded a revised version

**Comment by sgouezel** [comment]
bors r+
Thanks!

**Pull request closed without merging**